README.md

```diff
@@ -1,30 +1,32 @@
-# Project Migrator 
+# Project Migrator
 
 ### Requirements
 
-- You must have Deno installeds
+- You must have [Deno](https://deno.land/) installed. If you use Homebrew, run `brew install deno`.
 
 
 ### Considerations
 
-- These scripts (migrate and source) are supported strictly as is and LD support cannot be expected for help running this
+- These scripts, `migrate.ts` and `source.ts`, are provided strictly as-is. LaunchDarkly Support cannot be expected for help running this.
 
-### Known Issues
+### Known issues
 
 - Bug with 429 rate limiting. The request that is rate limited will still fail
-  but subsequent requests work
-- Importing LD API typescript types causes an import error so are commented out
-  in various spots
-- Types in generally are very loose which Deno is not happy about but it runs as
-  JavaScript overall compared to validating the TypeScript first
-- Due to the current API configuration, the environments count for a given project tops at 20 environments for a single project
-- Due to considerations around many API requests at once - monitor 400 errors for flag configurations that may not be up to date
-- To Avoid a race condition, a few waits have been placed in the script
+  but subsequent requests work.
+- Importing LD API TypeScript types causes an import error, so are commented out
+  in various spots.
+- Types in general are very loose, which Deno is not happy about. The scripts run as
+  JavaScript overall compared to validating the TypeScript first.
+- Due to the current API configuration, the environments count for a given project tops at 20 environments for a single project.
+- Due to considerations around many API requests at once, monitor 400 errors for flag configurations that may not be up to date.
```

Comment:
```suggestion
- Due to considerations around many API requests at once, monitor 400 errors for flag configurations that may not be up to date.
```
I wonder what we mean by "considerations" here.

Reply:
I think it's just another reference to the 429 (rate-limiting) errors.

Reply:
Yeah, we had an issue around some bad request types - the script will bubble up 400 errors for specific calls, but continue to run.  Wanted to highlight that the output is designed to be helpful, if things don't look right